tests/plugins/tableselection/manual/integrations/clipboard/selectall.md

```diff
@@ -0,0 +1,49 @@
+@bender-ui: collapsed
+@bender-tags: 4.13.1, bug, 875
+@bender-ckeditor-plugins: floatingspace, basicstyles, wysiwygarea, tableselection, clipboard, undo, selectall, blockquote, sourcearea, div, htmlwriter
+
+**Note:** As writing down all possible selections would make this test description really long,
+after conducting test steps have some fun pasting in other ways than described and look for bugs
+(e.g. using `div` plugin).
+
+However there is a known bug with <a href="https://github.com/ckeditor/ckeditor4/issues/541">disappearing table cells when mixed content is selected</a> 
+which is not a subject of this fix.
+
+## For each editor:
+
+1. Copy word `Start`, press <kbd>Ctrl+A</kbd> and paste.
+
+	### Expected:
+
+	The editor content have been replaced by pasted content.
+
+	### Unexpected:
+
+	Pasted data appeared in table, the rest of content is untouched.
+
+1. Undo changes.
+
+1. Select content from word `Before` to `After`.
+
+1. Paste copied content.
+
+	### Expected:
+
+	Selected content was replaced by pasted content.
+
+	### Unexpected:
+
+	Pasted data appeared in table, the rest of content is untouched.
+
+1. Undo changes.
+
+1. Create selection with content between brackets in table cell, paste.
```

Comment:
This step is a little unclear, I don't see any brackets in the table content (did you mean any of`([{` or something else here)?